modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/exp/agg/AccumulatorsFactory.java

```diff
@@ -300,15 +307,7 @@ private final class AccumulatorWrapperImpl implements AccumulatorWrapper<Row> {
             if (filterArg >= 0 && Boolean.TRUE != handler.get(filterArg, row))
                 return;
 
-            Object[] args = new Object[argList.size()];
-            for (int i = 0; i < argList.size(); i++) {
-                args[i] = handler.get(argList.get(i), row);
-
-                if (ignoreNulls && args[i] == null)
```

Comment:
Looks like we don't process ignoreNulls flag now. Shouldn't we process it by Function created by createInAdapter method?